Database/Constraint/Checks/Tent Checks.sql

```diff
@@ -25,17 +25,25 @@ GO
 /* Testdata */
```

Comment:
Style ontbreekt. Er staat bovenaan niet wie deze constraint heeft gemaakt, op welke dag en datum en wat de naam van de business rule is..

Even de Style aanvullen.